docs/network-flow-visibility.md

```diff
@@ -623,6 +623,7 @@ export.
 
 To achieve L7 (Layer 7) network flow export, the `L7FlowExporter` feature gate
 must be enabled.
+Note: L7 flow-visibility support for Theia is not yet implemented.
```

Comment:
Since this starts a new line, I suppose you mean to make it a separate paragraph? If yes, there should be an empty line before it in markdown.

Reply:
Yes, I've added an empty line. Thanks.